taeiim/MyApplication/app/src/main/java/com/god/taeiim/myapplication/ui/ContentsFragment.kt

```diff
@@ -32,24 +32,15 @@ class ContentsFragment : Fragment() {
         searchResultRecyclerView.adapter = this@ContentsFragment.adapter
 
         searchBtn.setOnClickListener {
-            searchContents(searchEditTv.text.toString())
+            searchEditTv.text.toString().let { presenter.searchContents(searchType.name, it) }
```

Comment:
```suggestion
searchBtn.setOnClickListener {
            presenter.searchContents(
                searchType.name,
                searchEditTv.text.toString()
            )
        }
```
이렇게 하는 것도 나쁘지 않을 것 같습니다.

Reply:
반영했습니다 감사합니다 ㅎㅎ ~